src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -285,6 +285,71 @@ internal static ulong DivRem(ulong a, ulong b, out ulong result)
             return div;
         }
 
+        [CLSCompliant(false)]
```

Comment:
Can we delete the internal methods above and switch callers to use the public ones?

Reply:
Done